helper/threads.c

```diff
@@ -144,10 +144,10 @@ static int create_process(odph_thread_t *thread, int cpu, uint64_t stack_size)
 /*
  * Wait single process to exit
  */
-static int wait_process(odph_thread_t *thread)
+static int wait_process(odph_thread_t *thread, odph_thread_join_result_t *res)
```

Comment:
As per current `odph_thread_join_result()` spec NULL is a valid value for `res`, this alone cannot be used to distinguish between `odph_thread_join()` and `odph_thread_join_result()`. Same for `wait_pthread()`.